### PR TITLE
Use persistent field in remember me

### PR DESCRIPTION
**Pull request by StanWarz**
[Asana ticket](https://app.asana.com/0/980131104682803/1156801986427609/f)

### CODE REVIEW

File: src/main/__tests__/authOptions.spec.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
-import { prepareAuthOptions } from '../authOptions'
+import { computeAuthOptions } from '../authOptions'
```
Comment (roxanemace) [inline src/main/__tests__/authOptions.spec.ts:1]: I don't see why `computeAuthOptions` is better than `prepareAuthOptions`.

File: src/main/apiClient.ts [hunk 2]
```diff
@@ -311,14 +311,17 @@ export default class ApiClient {
   }
 
   loginWithPassword(params: LoginWithPasswordParams): Promise<void> {
-    const saveCredentials = !isUndefined(params.saveCredentials) && params.saveCredentials
+    const storeCredentialsInBrowser = !isUndefined(params.saveCredentials) && params.saveCredentials
 
     const loginPromise =
-      window.cordova || saveCredentials
+      window.cordova || storeCredentialsInBrowser
```
Comment (roxanemace) [inline src/main/apiClient.ts:317]: Why did you change `saveCredentials ` into `storeCredentialsInBrowser`? It's different now from the parameter's name.

File: src/main/apiClient.ts [hunk 3]
```diff
@@ -141,9 +142,8 @@ export default class ApiClient {
         body: {
           clientId: this.config.clientId,
           grantType: 'authorization_code',
-          code: params.code,
-          redirectUri: params.redirectUri,
-          code_verifier: sessionStorage.getItem('verifier_key')
+          code_verifier: sessionStorage.getItem('verifier_key'),
```
Comment (roxanemace) [inline src/main/apiClient.ts:145]: This variable can be in camel case.

```suggestion
          codeVerifier: sessionStorage.getItem('verifier_key'),
```

File: src/main/apiClient.ts (hunk 2 above)
Comment (StanWarz) [reply src/main/apiClient.ts:317]: It's about documenting what this param aims for. I can also add a comment and rollback its name.

Comment (roxanemace) [reply src/main/apiClient.ts:317]: I fear it might be confusing to have 2 different variable names for the same value :/ I'd rather have a comment in the code (the parameter is already documented https://developer.reachfive.com/sdk-core/loginWithPassword.html#saveCredentials).

File: src/main/apiClient.ts (hunk 3 above)
Comment (StanWarz) [reply src/main/apiClient.ts:145]: I'm sorry you can't because it's the shape of expected body payload in endpoint `/oauth/token`
```private def tokenFromAuthorizationCode(
    client: Client,
  )(implicit req: RequestWithDomain[_],
  ): Future[Either[AppError, Map[String, Any]]] =
    trace("IdentityApi.tokenFromAuthorizationCode") {
      validateForm {
        Form(
          tuple(
            "code"          -> nonEmptyText,
            "redirect_uri"  -> nonEmptyText,
            "code_verifier" -> optional(nonEmptyText(43, 128)),
          ),
        )
      }.traverse {
        case (code, redirectUri, codeVerifier) =>
          authService.validateAuthCode(code, redirectUri, client, codeVerifier)
      }
    }```

File: src/main/__tests__/authOptions.spec.ts (hunk 1 above)
Comment (StanWarz) [reply src/main/__tests__/authOptions.spec.ts:1]: To me `prepareAuthOptions` means precreating it, so return an instance of `AuthOptions` while `computeAuthOptions` sounds to me building sth else from an instance of `AuthOptions` (here it's `AuthParams` though).

File: src/main/apiClient.ts (hunk 3 above)
Comment (roxanemace) [reply src/main/apiClient.ts:145]: The SDK Core transforms all the requests parameters into snake case https://github.com/ReachFive/identity-web-core-sdk/blob/master/src/main/httpClient.ts#L67
So it should be ok to convert the variable into camel case :)

File: src/main/apiClient.ts (hunk 2 above)
Comment (StanWarz) [reply src/main/apiClient.ts]: Updated as you required

File: src/main/apiClient.ts (hunk 3 above)
Comment (StanWarz) [reply src/main/apiClient.ts]: Alright, actually tested it and the payload indeed contains the snake_cased version of the param. Updated in the code.
